### PR TITLE
Added code for Electronics_Shop problem in java

### PR DESCRIPTION
**Pull request by Riyagoel2**
**Fixes issue:**

**Changes:**

### CODE REVIEW

File: code/online_challenges/src/hackerrank/Electronics_Shop/Electronics_Shop.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
+// Part of Cosmos by OpenGenus
+import java.io.*;
+import java.math.*;
+import java.text.*;
+import java.util.*;
+import java.util.regex.*;
+
+public class Solution {
```
Comment (arnavb) [inline code/online_challenges/src/hackerrank/Electronics_Shop/Electronics_Shop.java]: Reformat using 4 spaces.

**Comment by arnavb** [comment]
@Riyagoel2 Please reformat the file using 4 spaces.

File: code/online_challenges/src/hackerrank/Electronics_Shop/Electronics_Shop.java (hunk 1 above)
Comment (Riyagoel2) [reply code/online_challenges/src/hackerrank/Electronics_Shop/Electronics_Shop.java]: I changed it to 4 spaces but it automatically changes to 8 spaces.

Comment (arnavb) [reply code/online_challenges/src/hackerrank/Electronics_Shop/Electronics_Shop.java]: @Riyagoel2 What are you using? Try the IntelliJ code formatter.

Comment (Riyagoel2) [reply code/online_challenges/src/hackerrank/Electronics_Shop/Electronics_Shop.java]: I have made the changes .

**Pull request merged**